server/api/crud/artifacts.py

```diff
@@ -227,3 +250,49 @@ def _resolve_artifact_size(artifact, auth_info):
                     )
         if "spec" in artifact and "inline" in artifact["spec"]:
             validate_inline_artifact_body_size(artifact["spec"]["inline"])
+
+    def _delete_artifact_data(
+        self,
+        db_session: sqlalchemy.orm.Session,
+        key: str,
+        tag: str = "latest",
+        project: str = mlrun.mlconf.default_project,
```

Comment:
Don't use dynamic attributes as default value
```suggestion
        project: str = None,
       ..
       ...
       project = project or mlrun.mlconf.default_project
```